server/src/helpers/game/commands/gm/GMCreateGold.ts

```diff
@@ -0,0 +1,23 @@
+import { IMacroCommandArgs, IPlayer } from '../../../../interfaces';
+import { MacroCommand } from '../../../../models/macro';
+
+export class GMCreateGold extends MacroCommand {
+
+  aliases = ['@gold'];
+  isGMCommand = true;
+  canBeInstant = false;
+  canBeFast = false;
+
+  execute(player: IPlayer, args: IMacroCommandArgs) {
+    const gold = this.game.itemCreator.getSimpleItem('Gold Coin');
+    const value = +args.arrayArgs[0];
```

Comment:
This needs to do Math.round() so people can't make 1.1 gold.

Reply:
Thanks, did that as well as taking the absolute value of the entered number